### DISCUSSION TITLE
Confusion about smart_label: random in ShowCharts/MovieCharts?

### DISCUSSION THREAD

**Discussion by efaden** [meisnate12/Plex-Meta-Manager#252] [Q&A] [answered]
I have almost got it setup, but I wanted to see how the pre-made charts collections worked.  I see that it uses a template with a smart label for all of the collections.  I have read the smart_label section a few times, but I just don't get exactly what it is doing here?  Can someone shed some light?

**Comment by meisnate12** [comment, marked as answer]
with `smart_labels` PMM will add label tags to items instead of adding the item to a collection and then it will create a smart collection based on those labels.

**Reply by efaden** [reply]
So random just makes a random label, and then labels the movies with the label, and then makes a smart collection in plex looking for the labels?  I'm confused why this is advantageous.

**Reply by meisnate12** [reply]
The label is what the collection is called 

Random is the sort the collection will appear in

The main benefit is that is solves the problem the collectionless builder solved without having to have a collectionless collection

**Reply by efaden** [reply]
Gotcha.  Thanks!
